runtime/stdlib/builtin.go

```diff
@@ -121,6 +121,8 @@ var BuiltinFunctions = StandardLibraryFunctions{
 	CreatePublicKeyFunction,
 	AggregateBLSSignaturesFunction,
 	AggregateBLSPublicKeysFunction,
+	RLPDecodeStringFunction,
+	RLPDecodeListFunction,
```

Comment:
Keep naming consistent (for now):

```suggestion
	DecodeRLPStringFunction,
	DecodeRLPListFunction,
```